package.json

```diff
@@ -32,6 +35,8 @@
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
     "react": "^18.2.0",
+    "react-app-rewire-alias": "^1.1.7",
+    "react-app-rewired": "^2.2.1",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.5",
```

Comment:
all types, typescript and etc must be in dev dependencies or not?

Reply:
@au-e2xlabs 
May be)
If not difficult, could you change on your branch

Reply:
ok